src/api/app/controllers/webui/feeds_controller.rb

```diff
@@ -15,14 +16,9 @@ def latest_updates
   end
 
   def commits
-    @project = Project.find_by_name(params[:project])
-    if @project.nil?
-      render(file: Rails.root.join('public/404'), status: 404, layout: false, formats: [:html])
-      return
-    end
     # The sourceaccess flag is checked for the project, but not for every package
     if !User.current.is_admin? && @project.disabled_for?('sourceaccess', nil, nil)
-      render file: Rails.root.join('public/403'), formats: [:html], status: :forbidden, layout: false
+      redirect_to '/403.html', :status => :forbidden
```

Comment:
This was a problem in the controller test. It failed with some weird problem in ActionView::Resolver::Cache